commit fd33fd7c2674a9f02d8f47fa5bcc8321a9e5f694 (refs/changes/67/5467/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2019-02-01T16:24:35-08:00 (8 months ago)
    
    TRITON-1183 TRITON-805 has landed
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 10a0836..6a2a5cb 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "vm-agent",
     "description": "SmartDC Node VM Agent",
-    "version": "1.8.1",
+    "version": "1.8.2",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
@@ -18,7 +18,7 @@
         "restify-clients": "1.6.0",
         "tape": "4.2.2",
         "vasync": "1.6.3",
-        "vmadm": "https://github.com/joyent/node-vmadm.git#grr-TRITON-805"
+        "vmadm": "https://github.com/joyent/node-vmadm.git#v1.3.1"
     },
     "devDependencies": {
         "eslint": "4.19.1",
